numpy/core/numeric.py

```diff
@@ -2309,9 +2312,12 @@ def isclose(a, b, rtol=1.e-5, atol=1.e-8, equal_nan=False):
 
      absolute(`a` - `b`) <= (`atol` + `rtol` * absolute(`b`))
 
-    The above equation is not symmetric in `a` and `b`, so that
-    `isclose(a, b)` might be different from `isclose(b, a)` in
-    some rare cases.
+    Unlike the built-in `math.isclose`, the above equation is not symmetric
+    in `a` and `b` -- it assumes `b` is the reference value -- so that 
+    `isclose(a, b)` might be different from `isclose(b, a)`. Furthermore,
+    the default value of `atol` is not zero, and is targeted at numbers
+    of order unity; especially for numbers significantly smaller than one,
+    it can give false positives if not adjusted appropriately.
 
     Examples
     --------
```

Comment:
We should add an example or two (or four) that uses atol:

```
>>> np.isclose([1e-8, 1e-7], [0.0, 0.0])
array([ True, False], dtype=bool)

>>> np.isclose([1e-100, 1e-7], [0.0, 0.0], atol=0.0)
array([False, False], dtype=bool)

>>> np.isclose([1e-10, 1e-10], [1e-20, 0.0])
array([ True,  True], dtype=bool)

>>> np.isclose([1e-10, 1e-10], [1e-20, 0.999999e-10], atol=0.0)
array([False,  True], dtype=bool)
```

maybe an explanation of what these are demonstrating? or is is self explanatory enough?